cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureUtils.java

```diff
@@ -75,6 +75,8 @@ public class AzureUtils {
 
     public static final String DATABASE_PRIVATE_DS_ZONE_ID = "databasePrivateDsZoneId";
```

Comment:
Not introduced in this commit, but this one also seems to suffer from typos:
* `DS` -> `DNS`
* `Ds` -> `Dns`

Reply:
I am not sure I want to change it, as it serves as a key. If the resulting json is saved anywhere in DB then fixing a typo would create an error at readout time. I fixed it though for the aksPrivateDnsZoneId, however.

Reply:
I know it is a map key and would agree in other cases. I do no think the `Mappable` map representation is persisted anywhere, and neither is the `CloudStack` etc. whose dynamic parameters are populated using the aforementioned map. Anyway, we can keep it as is to be on the safe side.